portal/static/js/src/profile.js

```diff
@@ -1176,6 +1206,301 @@ export default (function() {
+                            }
+                            if ($(this).hasClass("other")) {
+                                answers.push({
+                                    "valueCoding": {
+                                        "code": $(this).attr("code"),
+                                        "system": `${location.origin}/api/codings/assessment`
+                                    },
+                                    "valueString": $(this).val()
+                                });
+                                let valueString = $(`#postTxQuestionnaireContainer .other-text[code="${$(this).attr("code")}"]`).val();
+                                if (valueString) {
+                                    answers.push({
+                                        "valueString": valueString
+                                    });
+                                }
+                                return true;
+                            }
+                            answers.push({
+                                "valueString": $(this).val()
+                            });
+                            answers.push({
+                                "valueCoding": {
+                                    "code": $(this).attr("code"),
+                                    "system": `${location.origin}/api/codings/assessment`
+                                }
+                            });
+                        }
+                        if ($(this).attr("dataType") === "boolean") {
+                            answers.push({
+                                "valueBoolean": $(this).is(":checked") ? true: false
+                            });
+                        }
+                    });
+                    answerSet.push({
+                        "answer": answers,
+                        "linkId": $(this).attr("linkId"),
+                        "text": $(this).attr("text")
+                    });
+                });
+                let patientReference = `${location.origin}/api/demographics/${this.subjectId}`;
+                postData.entry.push({
+                    "author":{
+                        "display":"user info",
+                        "reference":`${location.origin}/api/me/${this.currentUserId}`
+                     },
+                     "authored":this.modules.tnthDates.getTodayDateObj().gmtDate,
+                     "group": {
+                        "question": answerSet
+                     },
+                     "resourceType":"QuestionnaireResponse",
+                     "questionnaire":{
+                        "display":"EMPRO Post Intervention Questionnaire",
+                        "reference":`${location.origin}/api/questionnaires/${EMPRO_POST_TX_QUESTIONNAIRE_IDENTIFIER}`
+                     },
+                     "source": {
+                        "display": "patient demographics",
+                        "reference": patientReference
+                    },
```

Comment:
In the FHIR spec (see below), this refers to the person who answered the questions, and if I understand the story correctly, only staff will fill this out.

I don't think this is required so feel free to omit too


---
[`QuestionnaireResponse.source`](http://hl7.org/implement/standards/fhir/DSTU2/questionnaireresponse-definitions.html#QuestionnaireResponse.source):
>The person who answered the questions about the subject.